chia/server/address_manager.py

```diff
@@ -71,6 +79,52 @@ def to_string(self) -> str:
         )
         return out
 
+    @classmethod
+    def encode_ip_type(cls, ip: IPAddress) -> bytes:
+        if isinstance(ip._inner, IPv4Address):
+            return b"\x00"
+        elif isinstance(ip._inner, IPv6Address):
+            return b"\x01"
+        raise TypeError("Unsupported IPAddress type.")
```

Comment:
this might be reasonable to lable `pragma: no cover` as well, unless you can think of a test

Reply:
done